pages/Category.vue

```diff
@@ -1,125 +1,192 @@
 <template>
   <div id="category">
-    <header class="bg-cl-secondary py35 pl20">
-      <div class="container">
-        <breadcrumbs />
-        <div class="row middle-sm">
-          <h1 class="col-sm-8 category-title mb10">
-            {{ getCurrentCategory.name }}
-          </h1>
-          <div class="sorting col-sm-2 align-right mt50">
-            <label class="mr10">{{ $t("Columns") }}:</label>
-            <columns @change-column="columnChange" />
-          </div>
-          <div class="sorting col-sm-2 align-right mt50">
-            <sort-by
-              :has-label="true"
-              :value="getCurrentSearchQuery.sort"
-              @change="changeFilter"
-            />
-          </div>
-        </div>
+    <SfBreadcrumbs class="breadcrumbs desktop-only" :breadcrumbs="breadcrumbs">
+      <template #link="{breadcrumb}">
+        <router-link :to="breadcrumb.route.link">
+          {{ breadcrumb.text }}
+        </router-link>
+      </template>
+    </SfBreadcrumbs>
+    <div class="navbar section">
+      <div class="navbar__aside desktop-only">
+        <h1 class="navbar__title">Categories</h1>
       </div>
-      <div class="container">
-        <div class="row m0">
-          <button
-            class="col-xs-5 mt25 mr15 p15 mobile-filters-button bg-cl-th-accent brdr-none cl-white h5 sans-serif fs-medium-small"
-            @click="openFilters"
-          >
-            {{ $t("Filters") }}
-          </button>
-          <div class="mobile-sorting col-xs-6 mt25">
-            <sort-by
-              :value="getCurrentSearchQuery.sort"
-              @change="changeFilter"
-            />
-          </div>
+      <div class="navbar__main">
+        <SfButton class="navbar__filters-button">
+          <SfIcon size="15px" style="margin-right: 10px;">
+            <svg viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why we have the whole SVG code here? Can we create a separate component for this icon?

Reply:
Done - new folder for icons have been introduced: `src/themes/capybara/components/icons/<icon name.vue>`.